01-python/leisure/museum.py

```diff
@@ -1,3 +1,21 @@
-def draw_museum():
-    print("museum not found")
-    return
+def draw_museum(height = 3):
+	"""
+	Draw a museum with specified height.
+
+	Parameters:
+	height(int): the height of the pillars of the museum.
+
+	Returns:
+	int: the height of the pillars of the museum.
+
+	"""
+
+	print("   ==============")
+	print("  =              =")
+	print(" =                = ")
+	print("=                  =")
+	print("====================")
+	for _ in range(height):
+		print("|| || || || || || ||")
+	print("====================")
+	return height
```

Comment:
Just noticed this as I was about to merge. You should not return height here. Just return
```suggestion
	return
```